src/functions/usdclient.md

```diff
@@ -6,7 +6,16 @@ description: A compact funcion with 17 different functionalities.
 
 With this command you can get information about the bot user itself.
 
-Raw usage: `$client[option]`
+### Usage 
+```php
+$client[option]
+```
+
+### Fields
+
+| Field | Description | Type | Required |
+| :--- | :--- | :--- | :--- |
+| option | The option to be used | string | yes |
 
 #### Usable Options:
```

Comment:
eval $client[json]
and this should give you all available options

Reply:
Not working

Reply:
![image](https://user-images.githubusercontent.com/73595362/174431998-605603d5-00bf-495d-939b-893df99aecd2.png)
uh.....